app/models/points_manager.rb

```diff
@@ -7,8 +7,7 @@ def initialize(current_user, habit)
   def add_points
     habit_count = Dailyhabit.where(
       user_id: @user.id).where(
-      'date >= ?', DateTime.now.beginning_of_day..DateTime.now.end_of_day
-      ).count
+      'date >= ?', DateTime.now.beginning_of_day..DateTime.now.end_of_day).count
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.